framework/platform/bpl/CMakeLists.txt

```diff
@@ -47,11 +47,9 @@ if (TARGET_PLATFORM STREQUAL "openwrt")
         add_definitions(-DHAVE_C99)
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99")
 
-    elseif (TARGET_PLATFORM_TYPE STREQUAL "turris-omnia")
-
-        # Platform definitions
-        add_definitions(-DBEEROCKS_TURRIS_OMNIA)
+    else()
 
+        # Generic openwrt platform definitions
```

Comment:
The comment "Platform defintiions" was about the add_definitions statement, so this comment is not correct. Just remove it.

Reply:
Ok